File: linkedList.js
var LinkedNode = /** @class */ (function () {
    function LinkedNode(value) {
        this.value = value;
        this.next = null;
    }
    return LinkedNode;
}());
var a = new LinkedNode("A");
var b = new LinkedNode("B");
var c = new LinkedNode("C");
var d = new LinkedNode("D");
a.next = b;
b.next = c;
c.next = d;
d.next = null;
// A -> B -> C -> D -> null
var printLinkedList = function (head) {
    var current = head;
    while (current != null) {
        console.log(current.value);
        current = current.next;
    }
};
printLinkedList(a);

File: Linked_List.py
class Node:
    value = None
    next = None
    def __init__(self, value):
        self.value = value
        self.next = None

a = Node("A")
b = Node("B")
c = Node("C")
d = Node("D")

a.next = b
b.next = c
c.next = d
d.next = None

#A -> B -> C -> D -> None

def printLinkedList (head: Node = None):
    current = head
    while current is not None:
        print(current.value)
        current = current.next

printLinkedList(a)